fe2-android/app/src/main/java/com/github/dedis/popstellar/di/LAOModule.java

```diff
@@ -0,0 +1,49 @@
+package com.github.dedis.popstellar.di;
+
+import android.app.Application;
+
+import androidx.room.Room;
+
+import com.github.dedis.popstellar.repository.LAODataSource;
+import com.github.dedis.popstellar.repository.local.LAODatabase;
+import com.github.dedis.popstellar.repository.local.LAOLocalDataSource;
+import com.github.dedis.popstellar.repository.remote.LAORemoteDataSource;
+import com.github.dedis.popstellar.utility.scheduler.ProdSchedulerProvider;
+import com.github.dedis.popstellar.utility.scheduler.SchedulerProvider;
+
+import javax.inject.Singleton;
+
+import dagger.Binds;
+import dagger.Module;
+import dagger.Provides;
+import dagger.hilt.InstallIn;
+import dagger.hilt.components.SingletonComponent;
+
+@Module
+@InstallIn(SingletonComponent.class)
+public abstract class LAOModule {
```

Comment:
I think a better name would be `RepositoryModule` or `DataSourceModule`

Reply:
True, I will change that